discovery-provider/src/utils/helpers.py

```diff
@@ -96,6 +110,52 @@ def configure_logging(loglevel_str='WARN'):
     logger.setLevel(loglevel)
 
 
+def configure_flask_app_logging(app, loglevel_str):
+    # Disable werkzeug logger to remove default flask logs in favor of app decorator log below
+    werkzeug_log = logging.getLogger('werkzeug')
+    werkzeug_log.disabled = True
+
+    configure_logging(loglevel_str)
+    logger = logging.getLogger()  # retrieve root logger
+
+    # Set a timer before the req to get the request time
+    @app.before_request
+    def start_timer(): # pylint: disable=W0612
+        g.start = time.time()
+
+    # Log the request
+    @app.after_request
+    def log_request(response): # pylint: disable=W0612
+
+        now = time.time()
+        duration = int((now - g.start) * 1000)
```

Comment:
would this round down all durations of < 1 sec to 0?

Reply:
This should round request times to milliseconds

Reply:
ah misread this, all good